OpenRA.Game/FileSystem/ZipFile.cs

```diff
@@ -114,6 +114,9 @@ public ReadWriteZipFile(string filename, bool create = false)
 				pkgStream.Position = 0;
 				pkg = new ZipFile(pkgStream);
 				Name = filename;
+
+				foreach (ZipEntry entry in pkg)
```

Comment:
I'm sure that future maintainers would appreciate a comment here explaining why this is needed.